extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -49,22 +49,39 @@ amp-sticky-ad {
   display: block;
 }
 
-.-amp-sticky-ad-close-button {
+.amp-sticky-ad-close-button {
   position: absolute;
   display: block;
   height: 32px;
   width: 32px;
-  background-color: rgba(255,255,255,0.6);
-  color: grey;
-  font-size: 26px;
-  line-height: 26px;
-  vertical-align: middle;
+  background-color: #000;
+  opacity: 0.6;
   padding: 0;
-  top: -32px;
-  right: 0;
+  top: -34px;
```

Comment:
We might be able to be cleve here and use `transform: translate(0, -100%);`, which allows the box to always be the appropriate height above the ad regardless of text size.